lambda_src/response_storage.py

```diff
@@ -0,0 +1,47 @@
+import os
+from typing import Dict, Text
+
+import boto3
+
+AWS_REGION = os.environ[
+    'AWS_REGION'
+]
+DYNAMODB_TABLE = 'geff-requests'
+DYNAMODB_RESOURCE = boto3.resource('dynamodb', region_name=AWS_REGION)
+table = DYNAMODB_RESOURCE.Table(DYNAMODB_TABLE)
+
+
+def write_dynamodb_item(batch_id: Text, response: Dict):
+    """
+    Write to the requests table, a batch id and a response
+    """
+    table.put_item(Item={'batch_id': batch_id, 'response': response})
+
+
+def initialize_dynamodb_item(batch_id: Text):
+    """
+    Initialize an item in the requests table with a null response
+    """
+    table.put_item(Item={'batch_id': batch_id, 'response': None})
+
+
+def get_response(batch_id: Text):
```

Comment:
I think this will always be the same as the `check` function below, would it make sense to combine them?

Reply:
Yess, have only kept one get function now.